AK/JsonObject.h

```diff
@@ -113,6 +122,7 @@ class JsonObject {
     String to_string() const { return serialized<StringBuilder>(); }
 
 private:
+    HashMap<u32, String> m_order;
```

Comment:
Hmm, in the absence of something like an `OrderedHashMap` wouldn't this make more sense as a `Vector`?